tslint.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
As Page suggested, probably it's better to use `resin-lint` (and submit a PR if you think there should be some rules there that are enabled here).

Reply:
Dropping tsling.json :+1: 
It was ~ duplicating resin-lint's settings just to get SublimeText to provide a bit better linting.
Now let me find to configure my SublimeText linter to use the resin-lint's config...